src/ConsumeMemoryWithNoTask/ConsumeMemoryWithNoTask.csproj

```diff
@@ -0,0 +1,13 @@
+<Project Sdk="Microsoft.NET.Sdk">
+  <PropertyGroup>
+    <NoWarn>$(NoWarn);PolyfillTargetsForNuget</NoWarn>
+    <AllowUnsafeBlocks>true</AllowUnsafeBlocks>
+    <TargetFrameworks>netstandard2.0</TargetFrameworks>
+  </PropertyGroup>
+  <ItemGroup>
+    <!-- This project is specifically designed to guard against a mis-compile where System.Threading.Tasks.Extensions is referenced, but System.Memory is not. -->
+    <PackageReference Include="System.Threading.Tasks.Extensions" />
```

Comment:
Shouldn't these project names be reversed, i.e. this references Tasks but not Memory so it should be `ConsumeTaskWithNoMemory.csproj`?

Reply:
yep. good catch. fixed